setup.py

```diff
@@ -1,55 +1,9 @@
-import os
-import io
-import re
-from setuptools import setup, find_packages
+from setuptools import find_packages, setup
```

Comment:
Should we add copyright for all the python code in MONAI?

Reply:
thanks now fixed